src/olympia/devhub/templates/devhub/new-landing/components/manage_your_addon.html

```diff
@@ -6,15 +6,17 @@ <h3 class="DevHub-content-header DevHub-content-header--manage">{{ _('Manage You
     </div>
     <div class="DevHub-content-copy DevHub-content-copy--evolution">
       <div class="DevHub-callout-box DevHub-ext-ladder-icon">
-        <h3 class="DevHub-callout-header">{{ _('Help us improve the developer experience') }}</h3>
+        <h3 class="DevHub-callout-header">{{ _('Resources for a successful launch') }}</h3>
         <p class="DevHub-callout-copy">
           {% trans %}
-          We’re interested in your feedback about the extension development experience!
-          Take our short survey to let us know what kinds of documentation,
-          APIs, and tools would be most useful to your development work.
+          You’ve built your extension and are ready to publish it. 
+          What comes next? Check out our tips for creating an appealing
+          listing page, getting your extension featured on addons.mozilla.org,
```

Comment:
I would make `addons.mozilla.org` a link tbh, we tend to do it like this:

1. Define two variables only available in the `{% trans %} block that stands for the link opening and closing: ``{% trans amo_link_open='<a href="https://addons.mozilla.org/">'|safe amo_link_close='</a>' %}``
2. Use that variables in the template tag: ``{{ amo_link_open }}addons.mozilla.org{{ amo_link_close }}``